### PR TITLE
revert name from KEYFIFTHS to KEYSIGNATURE

### LINKED ISSUE #592: data_KEYSIGNATURE vs data.KEYFIFTHS
82e2648261eaf6afecc32e976789dd67fbab36bc seems to be part of an unfinished revision which we discussed in #525. Same with #590. 

I think we should revert these changes for 4.0.2 and (if we manage to clear things up) move them to 5.0.0.

### PR DESCRIPTION
**Pull request by rettinghaus**
fixes #592

### CODE REVIEW

File: source/modules/MEI.xml [hunk 1]
```diff
@@ -473,8 +473,7 @@
     </content>
   </macroSpec>
   <macroSpec ident="data.COLORNAMES" module="MEI" type="dt">
-    <desc>List of SVG-recognized color keyword names described at
-      https://www.w3.org/TR/SVG/types.html#ColorKeywords.</desc>
+    <desc>List of named colors from CSS Color Module Level 4.</desc>
```
Comment (bwbohl) [inline source/modules/MEI.xml]: This change actually doesn't belong here

File: source/modules/MEI.xml [hunk 2]
```diff
@@ -2567,8 +2575,8 @@
     </content>
     <remarks>
       <p>Instrument names are taken from the list at <ref
-        target="http://www.midi.org/techspecs/gm1sound.php"
-        >http://www.midi.org/techspecs/gm1sound.php</ref>. </p>
+        target="https://www.midi.org/specifications/item/gm-level-1-sound-set"
```
Comment (bwbohl) [inline source/modules/MEI.xml]: nothing to do with issue

File: source/modules/MEI.xml [hunk 3]
```diff
@@ -920,6 +922,12 @@
         </valItem>
       </valList>
     </content>
+    <remarks>
+      <p>Color names are taken from the list at <ref
```
Comment (bwbohl) [inline source/modules/MEI.xml]: nothing to do with issue

**Review by bwbohl — CHANGES_REQUESTED** [review]

**Comment by bwbohl** [comment]
All in all I'm not sure whether discussion of #525 and #590 and #592 are really at a point that can be considered decided once and for all. I don't like the idea of changing it back and forth. I think we have to come to a definite decision on those issues before merging.

**Comment by bwbohl** [comment]
Maybe we should even try to involve MEI-L on this?

File: source/modules/MEI.xml (hunk 1 above)
Comment (rettinghaus) [reply source/modules/MEI.xml]: something got mixed up

**Comment by rettinghaus** [comment]
@bwbohl changing it back and forth wasn't the idea here. The intention is to revert it, until a definite decision on all those matters has been made.

**Comment by rettinghaus** [comment]
@lpugin perhaps you want to approve this?

**Comment by lpugin** [comment]
@bwbohl ?

**Comment by kepper** [comment]
FWIW, I'm also not convinced to revert at this point. I'd like to have a plan before we take any further steps, even if they're only backwards…

**Pull request closed without merging**